README.md

```diff
@@ -1 +1,50 @@
 # HangOps Code of Conduct
+
+To continue growing a safe, welcoming, and fair community, all HangOps members are required to comply with the following code of conduct. HangOps admins are responsible for ensuring that all community members follow this code within HangOps spaces.
+
+## The Short Version
+
+Be respectful of other people; respectfully ask people to stop if you are bothered; and if you can’t resolve an issue contact the admins. If you are unable to follow our Community Code of Conduct problem, it will be apparent and you may be asked to leave HangOps.
```

Comment:
(I know this is the short version, but I have concern with some word choices)

> Be respectful 

It may be helpful to define `respect` since many folks probably have different meanings for it. Some folks often equate obedience and power to respect.

> bothered

This choice of word can be used in a way that blames the victim. 

> you are unable to follow 

Consider stronger wording. "If you do not follow"
> If you are unable to follow our Community Code of Conduct problem

Typo maybe? The word 'problem' makes the sentence structure seem weird to me. If I remove it, it reads more correctly to me.

> it will be apparent 

apparent to who?

> you may be asked 

'may be asked' may be too soft of a phrasing. What about stronger phrasing, "you will be ejected/banned from HangOps"